pytac/data_source.py

```diff
@@ -172,11 +169,9 @@ def get_value(self, field, handle=pytac.default, units=pytac.default,
             DeviceException: if there is no device on the given field.
             FieldException: if the manager does not have the specified field.
         """
-        if handle is pytac.default:
-            handle = self._default_handle
-        if units is pytac.default:
+        if units is pytac.DEFAULT:
             units = self._default_units
-        if data_source is pytac.default:
+        if data_source is pytac.DEFAULT:
```

Comment:
You should really use `==` not `is` in this case. I think it's unlikely to cause you a problem, but there's a chance that you end up with a string `default` that won't compare correctly as `DEFAULT`.

https://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce

Reply:
That stack overflow makes sense, interestingly I currently cause that problem in our code, but I will change it just in case.